components/sections/blogs/Posts.jsx

```diff
@@ -1,17 +1,17 @@
 /* eslint-disable react-hooks/exhaustive-deps */
 import { useEffect, useRef, useState } from "react";
-import Blog from "../ui/Blog";
-import Layout from "../layout/Layout";
+import Layout from "../../layout/Layout";
 import blogData from "/utils/blogData";
-import RightArrow from "../vectors/Arrow";
+import RightArrow from "../../vectors/Arrow";
+import RenderPosts from "./RanderPosts"
```

Comment:
Can you fix the **RanderPosts**, there is a typo. it should be `RenderPost`

Reply:
Nice